README.md

```diff
@@ -53,18 +52,39 @@ make install
 This will install to system default installation directory. If you want to
 change this location, use `-DCMAKE_INSTALL_PREFIX` option when running CMake.
 
-If you want to run tests, just type `ctest` in your build directory.
+If you want to build and run tests, in the build directory run:
+
+```
+cmake -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON ..
+ctest
+```
+
+# Building and installing on Windows
+
+To build `mgclient` on Windows MINGW environment could be used.
+   - Install MSYS2 from https://www.msys2.org/.
+   - Install MinGW toolchain with the following command:
+     ```
+     pacman -S --needed git base-devel mingw-w64-i686-toolchain mingw-w64-x86_64-toolchain mingw-w64-i686-cmake mingw-w64-x86_64-cmake mingw-w64-i686-openssl mingw-w64-x86_64-openssl
```

Comment:
pacman?

Reply:
https://www.msys2.org uses `pacman` as a package manager: "_To provide easy installation of packages and a way to keep them updated it features a package management system called Pacman, which should be familiar to Arch Linux users._" :smile: